raitools/raitools/_managers/explainer_manager.py

```diff
@@ -126,13 +158,55 @@ def name(self):
         :return: The name of the explainer manager.
         :rtype: str
         """
-        return "explain"
+        return ManagerNames.EXPLAINER
 
     def save(self, path):
-        raise NotImplementedError(
-            "Save not implemented for ExplainerManager")
+        """Save the ExplainerManager to the given path.
+
+        :param path: The directory path to save the ExplainerManager to.
+        :type path: str
+        """
+        top_dir = Path(path)
+        # save the explanation
+        if self._explanation:
+            save_explanation(self._explanation, top_dir / EXPLANATION)
+        meta = {IS_RUN: self._is_run,
+                IS_ADDED: self._is_added}
+        with open(Path(path) / META_JSON, 'w') as file:
+            json.dump(meta, file)
 
```

Comment:
We should also store the config of the explanation manager. Or since explanation manager is one static onfig it doesn't matter. I guess other components will have to do that as well. So maybe add a comment here to recover the configs in the respective managers.

Reply:
discussed, not needed in explainer_manager, but seems like a great idea for other explainers